apps/prairielearn/src/ee/auth/azure/callback.ts

```diff
@@ -1,13 +1,12 @@
-// @ts-check
-const passport = require('passport');
-const express = require('express');
-const asyncHandler = require('express-async-handler');
+import * as passport from 'passport';
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
```

Comment:
This weird little construct is necessary to import modules that look like `module.exports =`, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require. We'll be able to just use `import asyncHandler from 'express-async-handler'` when we start using native ESM.

Reply:
Ok, cool.